charts/sequencer/files/cometbft/config/config.toml

```diff
@@ -456,7 +456,7 @@ psql-conn = ""
 # When true, Prometheus metrics are served under /metrics on
 # PrometheusListenAddr.
 # Check out the documentation for the list of available metrics.
-prometheus = false
+prometheus = true
```

Comment:
Should we have this be the value of `.Values.serviceMonitor.enabled`?

Side note that we need to figure out how we can get the init to work on second deployment to update this file we copy (or get rid of the copy ideally)